### ISSUE TITLE
Spoiler images not displayed

### ISSUE THREAD

**Issue by kumy** [cgeo/cgeo#5722]
##### Detailed steps causing the problem:
- open a cache with spoiler images
- slide to images tab

##### Actual behavior after performing these steps:
only images from the description (maybe logs too)

##### Expected behavior after performing these steps:
spoiler images should also been displayed

##### Version of c:geo used:
2016.05.03-5cac039 developer build

##### Is the problem reproducible for you?
Yes

##### System information:
```
Attach system information here if available (see c:geo menu -> About c:geo -> System)
```

##### Other comments and remarks:
(not at home for some days, can someone cross check please)
found that yeterday, and someone else reported to me the same bug.

**Comment by SammysHP** [comment]
Was also reported in Twitter.

**@mcamunez79:**
> @android_gc Is there any problem with viewing images in c:geo ? 
From a few days ago, I cannot see any spoiler images in c:geo.

**Comment by UniQP** [comment]
Should be fixed since 85875c2e7a08a28468b4246aa5274cb4e2fde656, see #5718.
